src/app/community-list-page/community-list-page.component.ts

```diff
@@ -0,0 +1,9 @@
+import { Component } from '@angular/core';
+
+@Component({
```

Comment:
Please add TypeDocs/comments to describe this Component.  All new classes should have basic TypeDocs please.